interp/builtin.go

```diff
@@ -719,6 +720,11 @@ func (r *Runner) printOptLine(name string, enabled bool) {
 }
 
 func (r *Runner) readLine(raw bool) ([]byte, error) {
+	if r.stdin == nil {
+		r.errf("can't read, there's no stdin\n")
```

Comment:
if we're returning an error, I don't think we want to print one as well

Reply:
Thanks for the blazing fast review!